### PR TITLE
Add script for adding id.kb.se uris

### PR DESCRIPTION
**Pull request by kwahlin**

### CODE REVIEW

File: whelktool/scripts/cleanups/2020/11/lxl-3340-add-idkbse-uris.groovy [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        data#>>'{@graph,1,@type}' = 'Topic' OR
+        data#>>'{@graph,1,@type}' = 'ComplexSubject' OR
+        data#>>'{@graph,1,@type}' = 'Temporal' OR
+        data#>>'{@graph,1,@type}' = 'Geographic' OR
+        data#>>'{@graph,1,@type}' = 'TopicSubdivision')
+        AND (
+        data#>>'{@graph,1,inScheme,@id}' = 'https://id.kb.se/term/barn' OR
+        data#>>'{@graph,1,inScheme,@id}' = 'https://id.kb.se/term/sao')
+        AND NOT (
+        data#>>'{@graph,0,mainEntity}' LIKE '%id.kb.se%')
```
Comment (jannistsiroyannis) [inline whelktool/scripts/cleanups/2020/11/lxl-3340-add-idkbse-uris.groovy:15]: Perhaps a nitpick, but isn't there an @id missing here? :)
If the string "id.kb.se" exists somewhere (else) in the mainEntity part of the record, you'll incorrectly exclude that record.

File: whelktool/scripts/cleanups/2020/11/lxl-3340-add-idkbse-uris.groovy [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        """
+
+selectBySqlWhere(where) { data ->
+    String librisUri = data.graph[1]['@id']
+    String idkbseUri = data.graph[1].inScheme['@id'] + '/' + data.graph[1].prefLabel
+
+    data.graph[1]['@id'] = idkbseUri
+    data.graph[1].sameAs += ['@id':librisUri]
```
Comment (jannistsiroyannis) [inline whelktool/scripts/cleanups/2020/11/lxl-3340-add-idkbse-uris.groovy:23]: Neat! I didn't know the += operator worked on lists! Thanks for that ;)

**Review by jannistsiroyannis — APPROVED** [review]
There's a nitpick comment, but other then that, looks fine!

File: whelktool/scripts/cleanups/2020/11/lxl-3340-add-idkbse-uris.groovy (hunk 1 above)
Comment (kwahlin) [reply whelktool/scripts/cleanups/2020/11/lxl-3340-add-idkbse-uris.groovy:15]: Right! Adding the '@id' doesn't make any difference in this particular case though.

File: whelktool/scripts/cleanups/2020/11/lxl-3340-add-idkbse-uris.groovy (hunk 2 above)
Comment (kwahlin) [reply whelktool/scripts/cleanups/2020/11/lxl-3340-add-idkbse-uris.groovy:23]: Me neither! I used it unintentionally first (kind of copy pasted your pseudo-code) and then realized it actually worked. So thank you :)

**Review by klngwll — APPROVED** [review]
LGTM!

**Pull request merged**